modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenOperation.java

```diff
@@ -141,6 +141,15 @@ public boolean isRestfulUpdate() {
     }
 
     /**
+     * Check if act as Restful update method
+     *
+     * @return true if act as Restful update method, false otherwise
+     */
```

Comment:
the comment is wrong :wink: this is what I came up with, but wasn't fast enough to create a PR

    /**
     * Check if body param is allowed for the request method
     *
     * @return true request method is PUT, PATCH or POST; false otherwise
     */

Reply:
ah, now I see it. I updated the PR accordingly